init-default.sh

```diff
@@ -21,6 +21,5 @@ set_key "SECRET_KEY"
 set_key "ENCRYPT_KEY"
 set_key "VALIDATE_KEY"
 
-mkdir $CURR_DIR/data
-mkdir $CURR_DIR/data/warcs/
+mkdir -p $CURR_DIR/data
```

Comment:
Did you mean this to be `mkdir -p $CURR_DIR/data/warcs/` ? Just wanted to check.. Webrecorder does autocreate the `warcs` directory anyway now (it might not have initially when this script was made).